docs/pim/create_product.md

```diff
@@ -0,0 +1,56 @@
+---
+description: Create new products or modify existing ones.
+---
+
+# Create products
+
+[Products](products.md#products) are a specific kind of [Content items](../content_management/content_items.md#content-items) that you use 
+to present your offer in the website, including product specification, and pricing.
+
+Individual products are instances of [product types](create_product_types.md#create-product-types).
```

Comment:
I'd merge these three sentences into one paragraph. To do it, remove empty lines between them.

Reply:
Ok, updated